client/rpc-servers/src/middleware.rs

```diff
@@ -24,6 +24,21 @@ use prometheus_endpoint::{
 	U64,
 };
 
+/// Histogram time buckets in seconds.
+const HISTOGRAM_BUCKETS: [f64; 11] = [
+	5.0,
```

Comment:
5 sec as smallest bucket?  
Not rather milli-seconds? @bakhtin

Reply:
sorry the comment is wrong it's in micro seconds